tests/selection.spec.ts

```diff
@@ -649,3 +650,78 @@ test('the cursor should move to closest editor block when clicking outside conta
   await page.keyboard.press('Backspace');
   await assertRichTexts(page, ['123', '45', '789']);
 });
+
+test('should not crash when mouse over the left side of the list block prefix', async ({
+  page,
+}) => {
+  await enterPlaygroundRoom(page);
+  await initEmptyParagraphState(page);
+  await initThreeLists(page);
+  await assertRichTexts(page, ['123', '456', '789']);
+
+  await dragBetweenIndices(page, [1, 2], [1, 0]);
+  await copyByKeyboard(page);
+  await assertClipItems(page, 'text/plain', '45');
+  // blur
+  await page.mouse.click(0, 0);
+
+  await dragBetweenIndices(
+    page,
+    [1, 2],
+    [1, 0],
+    { x: 0, y: 0 },
+    { x: -50, y: 0 }
```

Comment:
why 50???

Reply:
<img width="660" alt="image" src="https://user-images.githubusercontent.com/14026360/216172949-2e42a7fa-9617-4ee4-83e4-34ee11d253e0.png">

the target position is on red dot

Reply:
see https://github.com/toeverything/blocksuite/pull/949